configs/components/cpp-pcp-client.rb

```diff
@@ -2,9 +2,13 @@
   pkg.load_from_json('configs/components/cpp-pcp-client.json')
   cmake = "/opt/pl-build-tools/bin/cmake"
   toolchain = "-DCMAKE_TOOLCHAIN_FILE=/opt/pl-build-tools/pl-build-toolchain.cmake"
-  pkg.environment "PATH" => "#{settings[:bindir]}:/opt/pl-build-tools/bin:$$PATH"
+  pkg.environment "PATH" => "#{settings[:bindir]}:#{settings[:tools_root]}/bin:$$PATH"
```

Comment:
I'm not sure. This is carried forward from the previous setup. I'd rather keep it around then take it out at this point in time.